mcp-servers/mcp-server-filesystem-edit/mcp_server_filesystem_edit/tools/helpers.py

```diff
@@ -4,7 +4,9 @@
 from typing import AbstractSet, Literal
 
 import tiktoken
+import tiktoken_ext  # Importing explicitly to get around pyinstaller issue
 from mcp_extensions.llm.llm_types import MessageT
+from tiktoken_ext import openai_public  # Importing explicitly to get around pyinstaller issue
```

Comment:
Instead of injecting imports in the code as a work around for packaging, you add details to the pyinstaller spec for "hidden imports".